### PR TITLE
Issues #158, #157, #153

### PR DESCRIPTION
**Pull request by MarionTheBull**
This PR corrects 3 issues. A bug with the argparse not being required, creation of an initial FAQ and bugs releated to salt formula collisions.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+#### What do I do if watchmaker failed to install?
+
+First, review the installation documentation provided with the
+watchmaker package. If pip is not working as your primary installation
+process you can attempt to install the package by cloning the repo of
+watchmaker.
+```python setup.py install```
+
+
+#### How do I know if watchmaker has completed with out errors?
```
Comment (KevinPlus3) [inline docs/faq.md]: Should be no space between "with out".

Comment (MarionTheBull) [inline docs/faq.md]: thanks.

File: docs/faq.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+the changes can take affect. If this happens watchmaker was executed
+entirely. After the reboot, the logs should be checked for errors.
+
+
+#### What do I do if watchmaker failed to complete or complete with errors?
+
+Start by checking the logs generated by watchmaker, defined by the
+argument that was passed to the watchmaker command when executing
+`--log-dir=/`. Search the log for entries that have `[ERROR]`, this will
+give you a starting point to begin troubleshooting. If its not an
```
Comment (KevinPlus3) [inline docs/faq.md]: "If it's not an obvious or simple issue, feel free to create ..."

**Comment by lorengordon** [comment]
Would be good to add a note to CHANGELOG.md, as well, for the behavior change as far as where the salt formulas end up.

**Pull request merged**